Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/data/FirebaseDatabase.kt

```diff
@@ -61,11 +61,19 @@ open class FirebaseDatabase {
             val socialProfiles: HashMap<String, String>? = HashMap(0),
             val pictureUrl: String = "",
             var bio: String = "",
-            var name: String = "") {
+            var name: String = "")
 
-    }
+    data class VolunteerEvent(
+            val twitter: String? = "",
+            val email: String = "",
```

Comment:
email is also nullable in the data but should be non-null if you initialize it like this

Reply:
I am doing a check later. Ideally, twitter should also be non-null. I check for empty. Avoiding nulls 😅